lib/core-data/get-variations.js

```diff
@@ -10,23 +10,35 @@ const log = logger(__filename);
  * @returns {object}
  **/
 function addField(schema, componentVariations) {
-  const variations = _.map(componentVariations, function (variant) {
+  // making the assumption that the component variation name is prefixed by the
+  // component name
+  const componentName = componentVariations[0].split('_')[0],
```

Comment:
what happens if/when this errors?

Reply:
`amphora-html` depends on this convention to figure out what component has variations, it then passes this data to Kiln. If this convention is broken, then that component will not have variations and this logic will not be run.

Reply:
What happens if there are no variations in the store (e.g. this version of Kiln is running against an older, non-variation version of amphora)? If it's taken into account already, please add a comment to that effect.